runner/tests/views/test_run_api_view.py

```diff
@@ -71,13 +71,13 @@ def test_tag(self):
         url = self.api_root + '?tags=tag:value'
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.json()['count'], 2)
```

Comment:
@sivkovic @nikhil @stevekm 

Because I've added an extra run, pipeline, operator, and operator run to the common fixtures, these numbers in the api view test have gone up, is this ok or is there a better way to do this?

Reply:
looks fine to me